src/utils/data.ts

```diff
@@ -114,8 +114,17 @@ export const extractDataSchema = async (
     if (Object.prototype.hasOwnProperty.call(data, key)) {
       const value = data[key];
       const typeOfValue = typeof value;
-      if (value instanceof Uint8Array) {
-        const guessedTypes = filetypeinfo(value);
+      if (value instanceof Uint8Array || value instanceof ArrayBuffer) {
+        let guessedTypes: Array<{
+          mime?: string;
+          extension?: string;
+          typename: string;
+        }> = [];
+        if (value instanceof Uint8Array) {
+          guessedTypes = filetypeinfo(value);
+        } else {
+          guessedTypes = filetypeinfo(new Uint8Array(value));
+        }
```

Comment:
Take `ArrayBuffer` into account:
Have a schema like `file: "application/octet-stream"` instead of `file: {}`